### ISSUE TITLE
follow_user_followers interract not working

### ISSUE THREAD

**Issue by alxhoff** [InstaPy/InstaPy#4787]
I am hoping to use the follow_user_followers in a loop to allow me to generate comments that include the account's name which I am targeting. The problem is that the interacting does not seem to be working. In my `set_comments` function only `session.set_comments` is performed to overwrite the current comment list.

## Expected Behavior
Instapy will comment and like one image from each follower that it follows
## Current Behavior
The followers are being followed but not interacted with
## InstaPy configuration
```
 314     session.set_do_comment(True, percentage=100)                                 
 315     session.set_do_like(True, percentage=100)                                    
 316                                                                                  
 317     for i in range(3):                                                           
 318         index = randint(0, len(influencers) - 1)                                 
 319         set_comments(session, False, influencers[index])                         
 320         session.set_user_interact(amount=1, randomize=True, percentage=100, media='Photo')
 321         session.follow_user_followers(influencers[index], amount=20, randomize=False, interact=True)
```

**Comment by converge** [comment]
could you check if this is working on `dev` branch? I fixed it today

**Comment by alxhoff** [comment]
Having problems on dev as use_firefox seems to of been removed and I am currently ahead of the latest chrome driver. Do you have a patch or PR for this particular fix?

**Comment by converge** [comment]
@alxhoff how do I reproduce the `use_firefox` issue?

**Comment by alxhoff** [comment]
@converge Just use the use_firefox=True argument when creating your session 
[https://github.com/timgrossmann/InstaPy/blob/master/DOCUMENTATION.md#switching-to-firefox](url)

**Comment by converge** [comment]
@alxhoff thats not necessary anymore, firefox is the default browser now (dev branch).

**Comment by alxhoff** [comment]
Ahh cool.

Now running in normal mode I am getting the selenium error 

> selenium.common.exceptions.WebDriverException: Message: invalid argument: can't kill an exited process

but more importantly when running headless. In util.py the lines like 2378 for example should be logger.logger.name instead of logger.name as the logger is being passed in as a LoggerAdapter and thus has no name. This is the same at login_util.py:122.

After resolving those it prompts me for the security code. After entering it I am receiving a second code sent to me, followed by the error:

CRITICAL [2019-08-07 17:26:35] [XXX]  Unable to login to Instagram! You will find more information in the logs above.

I applied the fix mentioned in #4598 but it didn't help. The trace is below.

Any ideas about the error popping up when not using headless?

> Traceback (most recent call last):
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/connection.py", line 159, in _new_conn
>     (self._dns_host, self.port), self.timeout, **extra_kw)
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/util/connection.py", line 80, in create_connection
>     raise err
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/util/connection.py", line 70, in create_connection
>     sock.connect(sa)
> ConnectionRefusedError: [Errno 111] Connection refused
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/connectionpool.py", line 600, in urlopen
>     chunked=chunked)
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/connectionpool.py", line 354, in _make_request
>     conn.request(method, url, **httplib_request_kw)
>   File "/usr/lib/python3.5/http/client.py", line 1106, in request
>     self._send_request(method, url, body, headers)
>   File "/usr/lib/python3.5/http/client.py", line 1151, in _send_request
>     self.endheaders(body)
>   File "/usr/lib/python3.5/http/client.py", line 1102, in endheaders
>     self._send_output(message_body)
>   File "/usr/lib/python3.5/http/client.py", line 934, in _send_output
>     self.send(msg)
>   File "/usr/lib/python3.5/http/client.py", line 877, in send
>     self.connect()
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/connection.py", line 181, in connect
>     conn = self._new_conn()
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/connection.py", line 168, in _new_conn
>     self, "Failed to establish a new connection: %s" % e)
> urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7fd78200f320>: Failed to establish a new connection: [Errno 111] Connection refused
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "smart_runtime.py", line 379, in <module>
>     standard_run()
>   File "smart_runtime.py", line 330, in standard_run
>     with smart_run(session):
>   File "/usr/lib/python3.5/contextlib.py", line 59, in __enter__
>     return next(self.gen)
>   File "/home/alxhoff/instabot/instapy/util.py", line 1839, in smart_run
>     session.login()
>   File "/home/alxhoff/instabot/instapy/instapy.py", line 400, in login
>     self.browser.implicitly_wait(temporary_page_delay)
>   File "/usr/local/lib/python3.5/dist-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
>     'implicit': int(float(time_to_wait) * 1000)})
>   File "/usr/local/lib/python3.5/dist-packages/selenium/webdriver/remote/webdriver.py", line 319, in execute
>     response = self.command_executor.execute(driver_command, params)
>   File "/usr/local/lib/python3.5/dist-packages/selenium/webdriver/remote/remote_connection.py", line 374, in execute
>     return self._request(command_info[0], url, body=data)
>   File "/usr/local/lib/python3.5/dist-packages/selenium/webdriver/remote/remote_connection.py", line 397, in _request
>     resp = self._conn.request(method, url, body=body, headers=headers)
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/request.py", line 72, in request
>     **urlopen_kw)
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/request.py", line 150, in request_encode_body
>     return self.urlopen(method, url, **extra_kw)
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/poolmanager.py", line 323, in urlopen
>     response = conn.urlopen(method, u.request_uri, **kw)
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/connectionpool.py", line 667, in urlopen
>     **response_kw)
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/connectionpool.py", line 667, in urlopen
>     **response_kw)
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/connectionpool.py", line 667, in urlopen
>     **response_kw)
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/connectionpool.py", line 638, in urlopen
>     _stacktrace=sys.exc_info()[2])
>   File "/usr/local/lib/python3.5/dist-packages/urllib3/util/retry.py", line 398, in increment
>     raise MaxRetryError(_pool, url, error or ResponseError(cause))
> urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=54164): Max retries exceeded with url: /session/aeec8555-f8a5-450c-bbec-c26a73a0cb00/timeouts (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd78200f320>: Failed to establish a new connection: [Errno 111] Connection refused',))
> ���

**Comment by alxhoff** [comment]
I am assuming it's something to do with two codes being sent and I am prompted to enter the first which is then invalidated by the second. Would be handy to be able to run not headless to see what's happening.
